app/js/referenceSearch.controller.js

```diff
@@ -1,6 +1,13 @@
-angular
-    .module('conceptDictionaryApp')
-    .controller('ReferenceSearchController', ['$scope', '$routeParams' ,'openmrsRest', function($scope, $routeParams, openmrsRest) {
+(function(){
+	'use strict'
+	
+	angular
+    	.module('conceptDictionaryApp')
+    	.controller('ReferenceSearchController', ReferenceSearchController)
+    	
+    ReferenceSearchController.$inject = ['$scope', '$routeParams' ,'openmrsRest', '$timeout']
+	
+	function ReferenceSearchController($scope, $routeParams, openmrsRest, $timeout){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many statements. (45)](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2330708537/issues/source?bid=3196969&fileBranchId=3207601#l10)